### ISSUE TITLE
Accessibility problems with the radio input

### ISSUE THREAD

**Issue by lud-hu** [formkit/formkit#1621]
### Reproduction

https://formkit.com/playground?fkv=latest&fileTab=radio-strings.vue&css-framework=genesis&files=jc%5B('name!'radio-strings.vue'~4!'%3Cscrip8setup3impor8(+ref+)+fromCvuD0cons8A5%3D+ref%7Bundefined%7D07script30%3C43%25target%25743'~editor!'GFormKit0**v-model6Ae9type6radio9label6Preferred+transportation9:options6%5B%22E-Bike1E-Scooter1Electric+car1Walking1Spac5tubD%5D9help6How+do+you+lik5to+ge8around?20*/3Gpr5wrap%3E((+A5))7pre3'~syntax!'html'~added!true)%5D*++0%5Cn1%22,C2%5C'3%3E04template5e+6%3D27%3C/8t+920**AvaluC+%22De%22G*%3C%01GDCA9876543210*_&imports=jc('name!'ImportMap'~lang!'json'~editor!'(*+1vue%5C!1https://cdn.jsdelivr.net/npm/vue@3/dist/vue.esm-browser.min.js0*)*')*%5Cn0%5C'1+0%0110*_

### Describe the bug

An accessibility issue with the FormKit radio component popped up during our accessibility audit. Radio groups should not use the unsorted list html elements, that FormKit is currently using. The proper roles (`radiogroup` and `radio`) should be assigned. See https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio-rating/

As a workaround one can use the sections schema to add the roles to the `<ul>` and `<li>` elements.
```html
  <FormKit
    v-model="value"
    type="radio"
    label="Preferred transportation"
    :options="['E-Bike', 'E-Scooter', 'Electric car', 'Walking', 'Space tube']"
    help="How do you like to get around?"
    :sections-schema="{
      options: {
        attrs: {
          role: 'radiogroup',
        },
      },
      option: {
        attrs: {
          role: 'radio',
        },
      },
    }"
  />
```
https://formkit.link/f9239e6c74ef8f4a4ee0aa5a3adace8a

### Environment

• OS: doesn't matter
• Browser: doesn't matter
• Version: doesn't matter
